demosplan/DemosPlanCoreBundle/Resources/client/scss/components/_tiptap.scss

```diff
@@ -250,8 +250,13 @@
                 font-weight: bold;
             }
 
-            ul,
+            ul {
+                list-style: disc;
+                padding-left: 1rem;
```

Comment:
Nitpick: back in the days it was a best practice to define a block with properties that are shared by multiple selectors first, followed by the blocks with the styles that are different, like so:

```scss
ul,
ol {
    // Things that ul and ol have in common
}

ul {
  // unique styles
}

ol {
  // unique styles
}
```

This is only to enhance readability, i think that the compiler will produce the same output for both versions.

Reply:
Since @sakutademos is out of office, I allowed myself to make the adjustment: 55d6767a308acac1dd9239df6e63ce9550f2ac87